tests/unit/test_client.py

```diff
@@ -1094,21 +1094,6 @@ def test_client_s3_addressing_style_path_registers_correctly(self):
             client.meta.events.register.call_args_list
         )
 
-    def test_create_with_ca_bundle_from_scoped_config(self):
```

Comment:
We talked about it, but the logic is now handled in the session's get_config_variable() method